atlp-rwanda/hackers-ec-Fe
# #187418993 Setting up redux toolkit to manage states globally.

# What does this PR do?
- This PR has setup of global state management
## Description
- I project there is a folder called redux
- The redux folder contains store file, hooks and features folder
- Store holds the global states of users I have declared 
- Hooks holds useDispatch and useSelector hooks to give access to the states in store and dispatching actions
- Feature contains the states of users to be managed. 

# How to test this?
- There is two pages in pages folder one is Users and other is OtherUsers
- You can check if the state of the Users is being handled in OtherUsers.
- You can also try to add another slice after userSlice to check if setup is working

**_Pivotal tracker ID_**
    
    #187418993
